test/Microsoft.Extensions.Caching.Memory.Tests/EntryLinkExpirationTests.cs

```diff
@@ -94,6 +94,72 @@ public void TokenExpires_LinkedEntry()
         }
 
         [Fact]
+        public void TokenExpires_ParentScopeEntry()
+        {
+            var cache = CreateCache();
+            var obj = new object();
+            string key = "myKey";
```

Comment:
Yeah, but if I do that then I have to change all unit tests in the whole package for consistency ;)